Formula/libosinfo.rb

```diff
@@ -27,6 +27,7 @@ def install
       --localstatedir=#{var}
       --mandir=#{man}
       --sysconfdir=#{etc}
+      --disable-dependency-tracking
       --disable-silent-rules
       --disable-udev
```

Comment:
`--disable-udev` can be dropped, it's no longer supported.

https://gitlab.com/libosinfo/libosinfo/commit/16cf1f797728dff9e361033937faea9f38ffbe49